account_user_audit/__openerp__.py

```diff
@@ -0,0 +1,60 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2004-2009 Tiny SPRL (<http://tiny.be>). All Rights Reserved
+#    d$
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": 'Add Read-Only Accounting Group',
+    "version": "0.1",
+    "author": "Vauxoo",
+    "category": "Generic Modules",
+    "description": """
+
+Overview:
+===============================
```

Comment:
@hugho-ad en RST esta linea de símbolos debe terminar a la par que el texto. No usar "Overview", usar algo como "This module adds the following features:" o "The following feature:"